biojava-structure/src/main/java/org/biojava/nbio/structure/symmetry/core/QuatSymmetryParameters.java

```diff
@@ -49,14 +49,9 @@
 	private double minimumHelixRise = 1.0;
 	private double minimumHelixAngle = 5.0; // min helix angle to differentiate
 											// it from a translational repeat
-	private int maximumLocalCombinations = 50000; // max number of combinations
-													// to try for local symmetry
-													// calculation
-	private int maximumLocalResults = 1000;
-	private int maximumLocalSubunits = 20; // maximum number of subunits for
-											// local symmetry calculations
 	private double localTimeLimit = 120; // time limit for local calculations in
```

Comment:
I am not sure that a time limit is a good idea here (this is how it was done before). 

I preferred calculating the number of possible combinations and only evaluating a subset of them if they were too large, since it allows to know before starting the calculation and we can use whatever heuristic we considered adequate to select the combinations we want. 
Also, it makes it easier to debug (in case a structure fails with too long runtime but very few combinations it would be hard to notice how the code is constructed now). In addition, runtime is very much system dependent and can make results less reproducible.

What is your logic to go back to using time limits to stop the calculation for large structures?